src/types/filingTypes.ts

```diff
@@ -143,8 +143,11 @@ export interface ValidationResults {
 }
 
 export interface ValidationErrorWarning {
-  count: number;
   details: Detail[];
+  multi_field_count: number;
```

Comment:
I think some of these changes are taken care of by @shindigira's PR already [over here](https://github.com/cfpb/sbl-frontend/pull/699/files#diff-580e104f64a806e9ed0e66e14eec71aa0d3f851a2ca9d7650ea3a14e4f63d28aR146-R149). I don't this his PR fixes the count on the submission page though. He didn't get a chance to merge in his PR before he left on vacation, so I'm going to go ahead and merge it now and you can update this branch and see what changes still need to be made @meissadia?